docs/uplink/installation.md

```diff
@@ -0,0 +1,177 @@
+# Install inlets-uplink
+
+Inlets Uplink is a self-hosted solution for managing many tunnels for a team or company.
+
+Learn why we created Inlets Uplink and how it might be a good fit for you in the announcement blog post:
+
+- [Inlets Uplink for SaaS & Service Providers](https://inlets.dev/blog/2022/11/16/service-provider-uplinks.html)
+
+## Pre-reqs
+* A Kubernetes cluster with LoadBalancer capabilities (i.e. public cloud).
+* A domain name clients can use to connect to the tunnel control plane.
+* [arkade](https://github.com/alexellis/arkade), a simple CLI tool that provides a quick way to install various apps and download common binaries much quicker
+    
+    To install arkade run:
+
+    ```bash
+    curl -sSLf https://get.arkade.dev/ | sudo sh
+    ```
+
+## Create a Kubernetes cluster
+
+Create a cluster with at least 3 nodes 2-4GB of RAM each
+
+## Deploy with Kubernetes Ingress
+
+### Install the IngressController
+
+This installs nginx-ingress using its Helm chart:
+
+```bash
+arkade install ingress-nginx
+```
+
+### Install cert-manager
+
+Install [cert-manager](https://cert-manager.io/docs/), which can be integrated with NginxIngress to manage TLS certificates.
+```bash
+arkade install cert-manager
+```
+
+### Deploy inlets-uplink with Helm
+
+Make sure to create the target namespace for you installation first.
+
+```bash
+kubectl create namespace inlets
+```
+
+Create the required secret with your inlets-uplink license.
+
+```bash
+kubectl create secret generic \
+  -n inlets inlets-uplink-license \
+  --from-file license=$HOME/.inlets/LICENSE_UPLINK
+```
+
+Create a `values.yaml` file for the inlets-uplink chart:
+
+```yaml
+clientRouter:
+  # Customer tunnels will connect with a URI of:
+  # wss://uplink.example.com/namespace/tunnel
+  domain: uplink.example.com
+
+  tls:
+    issuer:
+      # Email address used for ACME registration
+      email: "user@example.com"
+
+    ingress:
+      enabled: true
+      class: "nginx"      
+```
+
+Make sure to replace the domain and email with your acatual domain name and email address.
+
+Deploy inlets-uplink with your configuration values:
+
+```
+helm upgrade --install inlets-uplink \
+  oci://ghcr.io/inlets/inlets-uplink-operator \
+  --namespace inlets \
+  --values ./values.yaml
+```
+
+## Install with Istio
+We have added support in the inlets-uplink chart for Istio to make it as simple as possible to configure with a HTTP01 challenge.
+
+### Install Istio
+If you don't have Istio setup already you can deploy it with arkade.
+
+```bash
+arkade install istio
+```
+
+### Install cert-manager
+
+Install [cert-manager](https://cert-manager.io/docs/), which can be integrated with Istio gateways to manage TLS certificates.
```

Comment:
Cert manager is needed either way, so let's do it before giving A or B as options